<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dfslog</class>
 <widget class="QDialog" name="dfslog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="rand_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>随机生成图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_Button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_circle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>圆形排列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="last_Button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
